### DISCUSSION TITLE
Does crawlee-python has a task distributed or manage system like scrapyd and scrapyd-web?

### DISCUSSION THREAD

**Discussion by QThans** [apify/crawlee-python#679] [Q&A] [answered]

**Comment by vdusek** [comment, marked as answer]
Crawlee does not include native support for "daemonization" or task distribution, but you can integrate it with other libraries that provide these features. For instance, to run a Crawlee project as a daemon, you can embed it within a framework like [FastAPI](https://fastapi.tiangolo.com/) and communicate with it using HTTP. If you need distributed task management, [Celery](https://docs.celeryq.dev/en/stable/getting-started/introduction.html) could be an option, which can handle task scheduling, queuing, and scaling. Both FastAPI and Celery are free, open-source tools, that you can integrate with Crawlee.

If you’re interested in hosting your crawlers on a managed platform, you can use Apify. Check out our [Deploy to Apify](https://crawlee.dev/python/docs/deployment/apify-platform) guide for more details.

**Reply by QThans** [reply]
ok, thanks for your replay.
